AlpsSDK/Repositories/PubSub/PublicationRepository.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  PublicationRepository.swift
+//  AlpsSDK
+//
+//  Created by Maciej Burda on 27/10/2017.
+//  Copyright © 2017 Alps. All rights reserved.
+//
+
+import Foundation
+import Alps
+
+final class PublicationRepository: AsyncCreateable, AsyncReadable, AsyncDeleteable {
+    typealias DataType = Publication
+    private(set) var items = [Publication]()
```

Comment:
Each device has his own Pub/Sub repository ? 
or main device will keep all the publication from "session"

example : 
Main device creates a Pin device and attach a publication on it. 
Where will this publication be saved ??

Reply:
Take a look at ` func createPublication(publication: Publication, for deviceWithId: String? = nil, completion: ((Publication?) -> Void)? = nil)`

It's convenice method in `AlpsManager`.

All publications are stored in publication repostiory. If you want to pull those releated to device you need to filter by `deviceId`.

Reply:
got it, thanks